sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosEndToEndOperationLatencyPolicyConfig.java

```diff
@@ -13,19 +13,27 @@ public final class CosmosEndToEndOperationLatencyPolicyConfig {
     private final boolean isEnabled;
     private final Duration endToEndOperationTimeout;
 
+    private final AvailabilityStrategy availabilityStrategy;
+    private final Duration endToEndFeedOperationTimeout;
+
     /**
      * Constructor
      *
-     * @param isEnabled                toggle if the policy should be enabled or disabled
-     * @param endToEndOperationTimeout the timeout for request cancellation in {@link Duration}. Setting very low timeouts
-     *                                 can cause the request to never succeed.
+     * @param isEnabled                    toggle if the policy should be enabled or disabled
+     * @param endToEndOperationTimeout     the timeout for request cancellation in {@link Duration}. Setting very low timeouts
+     *                                     can cause the request to never succeed.
+     * @param endToEndFeedOperationTimeout
```

Comment:
I suppose the constructor parameters are still up for discussion - this parameter name can be modified to `endToEndNonPointOperationTimeout`